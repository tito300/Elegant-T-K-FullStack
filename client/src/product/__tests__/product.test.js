import { shallow } from 'enzyme';
import React from 'react';
import Product from '../product';

let wrapper;

describe('Product component', ()=>{
    beforeAll(()=>{
        let props = {
            product: {photo: 'img/random.jpg'}
        }
        wrapper = shallow(<Product {...props}/>);
    })
    it('should match Product snapshot', ()=>{
        expect(wrapper).toMatchSnapshot();
    }) 
})