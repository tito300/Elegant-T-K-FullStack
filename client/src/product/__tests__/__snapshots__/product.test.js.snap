// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product component should match Product snapshot 1`] = `
<div
  className="productSubContainer"
>
  <div
    className="TopSection"
  >
    <ProductImage
      activePicture="img/random.jpg"
      product={
        Object {
          "photo": "img/random.jpg",
        }
      }
    />
    <div
      className="infoSection"
    >
      <ProductDetails
        activePicture="img/random.jpg"
        addItemToCartSuccess={[Function]}
        color={null}
        gender="male"
        handleOptions={[Function]}
        product={
          Object {
            "photo": "img/random.jpg",
          }
        }
        size={null}
        success={false}
        variants={null}
      />
      <_default
        color={null}
        gender="male"
        handleOptions={[Function]}
        size={null}
      />
    </div>
  </div>
  <div
    className="BottomSection"
  >
    <DisplaySimilarProducts
      product={
        Object {
          "photo": "img/random.jpg",
        }
      }
    />
  </div>
</div>
`;
